src/getHumanAge.test.js

```diff
@@ -7,4 +7,48 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return an array with 2 elem', () => {
+    expect(getHumanAge(1, 5))
+      .toHaveLength(2);
+  });
+
+  test(`should return an the array where
+- the first element is my cat's age in **human** years
+- the second element is my dog's age in **human** years`, () => {
+    expect(getHumanAge(5, 40))
+      .toEqual([0, 5]);
+  });
+
+  test(`should return 0 if the years are 0 `, () => {
```

Comment:
```suggestion
  test(`should return [0, 0] if the years are (0, 0) `, () => {
```
smth like that, because now it seems like the function returns a number and accepts one number, but it's not true